ckanext/canada/tables/travelq.yaml

```diff
@@ -187,37 +187,189 @@ resources:
   - datastore_id: destination_en
     datastore_type: text
     label:
-      en: Places visited (English)
+      en: Place visited (English)
       fr: Endroits visités (anglais)
     description:
-      en: To include names of all places visited during the trip in question, in English
+      en: To include name of the place visited during the trip in question, in English.
       fr: Cette zone indique, en anglais, le nom de tous les endroits visités pendant le voyage.
+    form_help:
+      en: Note, use the format {City Name}, {Country Name} (e.g. Ottawa, Canada or New York City, United States of America)
+      fr: Note, use the format {City Name}, {Country Name} (e.g. Ottawa, Canada or New York City, United States of America)
     obligation: Mandatory
     excel_required: true
```

Comment:
this is close: check for one comma and something non-whitespace on either side
```suggestion
    excel_required: true
    excel_error_formula: 'OR({default_formula},LEN({cell})<>LEN(SUBSTITUTE({cell},",","")+1,LEN(TRIM(LEFT({cell},FIND(",",{cell})-1)))>0,LEN(TRIM(RIGHT({cell},LEN({cell})-FIND({cell},","))))>0)
```

Reply:
@wardi `err:508` in excel on this

Reply:
I'm sure I lost my way pasting in some of the parens or something. I'll take another look

Reply:
Try it again now

Reply:
still broken sadly. I worked on these formulas in different ways for like 6 hours and couldn't get them. So ultimately gave up and easily convinced people to drop the parity between PSQL and Excel

Reply:
@JVickery-TBS this one works with either 1 or 2 commas
![excel_brain_damage](https://github.com/user-attachments/assets/8af9bf12-cd60-4d70-86a8-3c9fa5d25de3)

Reply:
`{1,2}` will cause error in python as Recombinant formats the formulae. Ima just executive decision to not do excel error formulas for these fields and call it a day.

Reply:
right, there's a format() in there. I just doubled them up to fix.

Reply:
there is actually also an assertion in Recombinant which forces the format values to be exact, so everything needs a replacement. We are just not gunna support it here with this field. And I doubt we could get a formula for the multi-destination field either